### PR TITLE
Use conda-forge/websockify, use environment.yml in Dockerfile

### PR DESCRIPTION
**Pull request by manics**
websockify was accepted into conda-forge 🙂 https://github.com/conda-forge/staged-recipes/pull/12168

This PR also reduces duplication by copying `environment.yml` into the `Dockerfile`

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
...
-  - manics
+  - conda-forge
 dependencies:
+  - jupyter-server-proxy>=1.4
+  - pip
   - websockify
   - pip:
-    - .
\ No newline at end of file
+      - jupyter-desktop-server==0.1.3
```
Comment (yuvipanda) [inline environment.yml]:
```suggestion
      - .
```

I think this should still be '.', so it's running the latest version in the repo.

**Comment by yuvipanda** [comment]
This is awesome, thank you :) One comment, otherwise great to merge.

File: environment.yml (hunk 1 above)
Comment (manics) [reply environment.yml]: I had to update the Dockerfile as well, see the last commit

**Comment by yuvipanda** [comment]
Awesome, thanks @manics!

**Pull request merged**